modules/periphery/metadata.json

```diff
@@ -11,9 +11,11 @@
         "github:peripheryapp/periphery"
     ],
     "versions": [
-        "3.0.0.beta7",
```

Comment:
Please keep these lines so that the module docs in the registry UI don't vanish. They will be marked as yanked.